app/models/post.rb

```diff
@@ -1,25 +1,11 @@
 class Post < ActiveRecord::Base
-  belongs_to :user
   has_and_belongs_to_many :tags
-  has_many :comments, as: :parent
+  validates_presence_of :title
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.